microsoft-iis-mixin/dashboards/microsoft-iis-overview.libsonnet

```diff
@@ -0,0 +1,1284 @@
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      decimals: 0,
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+          {
+            color: 'red',
+            value: 80,
+          },
+        ],
+      },
+      unit: 'errors/sec',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'single',
+      sort: 'none',
+    },
+  },
+};
+
+local blockedAsyncIORequestsPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'increase(windows_iis_blocked_async_io_requests_total{job=~"$job", instance=~"$instance", site=~"$site"}[$__interval:])',
+      datasource=promDatasource,
+      legendFormat='{{instance}} - {{site}}',
+      interval='1m',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Blocked async I/O requests',
+  description: 'Number of async I/O requests that are currently queued for an IIS site.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 0,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        lineInterpolation: 'linear',
+        lineWidth: 1,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'auto',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      decimals: 0,
+      mappings: [],
+      min: 0,
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+          {
+            color: 'red',
+            value: 80,
+          },
+        ],
+      },
+      unit: 'requests',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'none',
+      sort: 'none',
+    },
+  },
+};
+
+local rejectedAsyncIORequestsPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'increase(windows_iis_rejected_async_io_requests_total{job=~"$job", instance=~"$instance", site=~"$site"}[$__interval:])',
+      datasource=promDatasource,
+      legendFormat='{{instance}} - {{site}}',
+      interval='1m',
```

Comment:
any specific reason to fix min interval here?

Reply:
Yes, there has been some discussion on why we use this 1m interval with `__interval` a good explanation would stem from this thread on the previous [Wildfly mixin](https://github.com/grafana/jsonnet-libs/pull/953#issuecomment-1481851319). It essentially prevents some data inconsistencies coming through when using the `increase` function.

Reply:
@v-zhuravlev most of this discussion was in the Slack thread where we decided (for now) to use `$__interval:` and `inteval: 1m` for increase functions as a standard